src/main/java/errorMessages/ResponseErrorMessage.java

```diff
@@ -0,0 +1,7 @@
+package errorMessages;
+
+public interface ResponseErrorMessage {
+    String NOT_FOUND = "Not Found";
```

Comment:
если ты хочешь четко определить константы, то поля лучше писать как
```suggestion
    public static final String NOT_FOUND = "Not Found";
```